packages/terra-functional-testing/package.json

```diff
@@ -69,27 +69,5 @@
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0",
     "which": "^2.0.2"
-  },
-  "devDependencies": {
```

Comment:
moved to root

Reply:
actually i think some of these dependencies should be moved to regular dependencies of this package.